content/_index.md

```diff
@@ -3,6 +3,8 @@
 
 ### News
 
+- **Conference 2025**: We are planning our second Nordic-RSE conference on May 20 + 21 2025 in Gothenburg, Sweden. Call for abstracts and registration opening soon.
```

Comment:
optional: should we link to thte conference website? now it has basic info and CFP already.

Reply:
If it is ready for that, then we definitely should!

Reply:
Let's link to it already. The most important thing are dates. Detailed info might shift a bit but I see now harm in linking to draft.